modin/experimental/cloud/rpyc_proxy.py

```diff
@@ -622,6 +629,27 @@ def iteritems(self):
     ObtainingItems = _deliveringWrapper(Series, mixin=ObtainingItems)
 
     class DataFrameOverrides(_prepare_loc_mixin()):
+        @classmethod
+        def _preprocess_init_args(
+            cls,
+            data=None,
+            index=None,
+            columns=None,
+            dtype=None,
+            copy=None,
+            query_compiler=None,
+        ):
+
+            (data,) = conn.deliver((data,), {})[0]
```

Comment:
I specified only `data` because if I add other parameters (except `copy` and `query_compiler`), errors may occur, the essence of which is not yet clear.